src/EntityFramework6.Npgsql/EntityFramework6.Npgsql.nuspec

```diff
@@ -16,7 +16,7 @@
     <language>en-US</language>
     <tags>npgsql postgresql postgres data database entity framework ef orm</tags>
     <dependencies>
-      <dependency id="Npgsql" version="3.1.0" />
+      <dependency id="npgsql" version="3.1.0" />
       <dependency id="EntityFramework" version="6.0.0" />
```

Comment:
The new nuget CLI i.e. v. 3.5 that is used when you do ```dotnet restore``` does not do that if a specific version is requested and no other package wanted a higher version.